appveyor.yml

```diff
@@ -1,16 +1,51 @@
 version: 1.0.{build}
+os:
+- Visual Studio 2013
+# only run PR-CA on the VS 2015 OS - please change the scripts if this string changes
+- Visual Studio 2015
+
+install:
+  - git submodule update --init --recursive
+  - ps: |
+      Add-Type -AssemblyName System.IO.Compression.FileSystem
+      if (!(Test-Path -Path "C:\maven" )) {
+        (new-object System.Net.WebClient).DownloadFile(
+          'http://www.us.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.zip',
+          'C:\maven-bin.zip'
+        )
+        [System.IO.Compression.ZipFile]::ExtractToDirectory("C:\maven-bin.zip", "C:\maven")
+      }
+  - cmd: SET PATH=C:\maven\apache-maven-3.2.5\bin;%JAVA_HOME%\bin;%PATH%
+
+cache:
+  - C:\sonarqube\downloads -> appveyor.yml
+  - C:\users\appveyor\.m2 -> appveyor.yml
+
+configuration: Release
 
 environment:
   matrix:
-  - Configuration: Release
-  - Configuration: Debug
+  - SQ_Version: '4.5.6'
+  - SQ_Version: '5.2'  
+    PR_CA: '1'
```

Comment:
Only run PR-CA on one of the configurations. This is a workaround for the fact that AppVeyor does not support multiple configs per project, nor can we differentiate between runs in the same build (i.e. matrix runs) vs separate build runs.